src/main/drivers/pwm_output.c

```diff
@@ -255,7 +255,7 @@ void motorInit(const motorConfig_t *motorConfig, uint16_t idlePulse, uint8_t mot
 
 #ifdef USE_DSHOT
         if (isDigital) {
-            pwmDigitalMotorHardwareConfig(timerHardware, motorIndex, motorConfig->motorPwmProtocol);
+            pwmDigitalMotorHardwareConfig(timerHardware, motorIndex, motorConfig->motorPwmProtocol, motorConfig->motorPwmPolarity);
```

Comment:
I may want to use DSHOT with a brushed FC converted for brushless operation, and it will suffer from the same issue - inverted active level. I think it makes sense after all.